SECURITY.md

```diff
@@ -0,0 +1,18 @@
+# Security Policy
+
+## Supported Versions
+
+Security updates are currently supported for Pacemaker's 2.1 and 3.0 release
```

Comment:
Yeah I'm leery of the word "support" as well :) but upstream it has a different meaning. This is based mostly on GitHub's template, so I figured it will be what people are familiar with seeing, but we can tweak it :)